fusor-ember-cli/app/routes/openstack/overcloud.js

```diff
@@ -7,11 +7,11 @@ export default Ember.Route.extend({
       changedParams = [
         {
           name: 'Controller-1::NeutronPublicInterface',
-          value: this.get('controller.openStack.externalNetworkInterface')
+          value: this.get('controller.externalNetworkInterface')
         },
         {
           name: 'Controller-1::AdminPassword',
-          value: this.get('controller.openStack.overcloudPassword')
```

Comment:
This was using what was in the deployment plan, not what was in the form.  This field isn't using an alias like most of our data because of [this issue](https://github.com/fusor/fusor/blob/master/fusor-ember-cli/app/routes/deployment.js#L61)